src/openforms/pre_requests/clients.py

```diff
@@ -38,6 +38,9 @@ class PreRequestMixin:
     than just some aspects of it.
     """
 
+    enabled_plugins: ClassVar[list[str]] = []
+    """A list of plugin identifiers that will be called before preparing the request."""
+
```

Comment:
When for an empty default: I feel that having all the plugins applied just by subclassing this mixin is a bit "dangerous". Instead, subclasses explicitly set the plugins they want to be applied.

Reply:
I added more info about this in the registry docstring